src/energyplus/Test/Site_GTest.cpp

```diff
@@ -49,6 +49,6 @@ TEST_F(EnergyPlusFixture, ForwardTranslator_Site) {
   EXPECT_EQ(0.0, idf_site.getDouble(Site_LocationFields::Longitude).get());
   EXPECT_EQ(0.0, idf_site.getDouble(Site_LocationFields::TimeZone).get());
   EXPECT_EQ(0.0, idf_site.getDouble(Site_LocationFields::Elevation).get());
-  EXPECT_EQ("Suburbs", idf_site.getString(Site_LocationFields::Terrain).get());
+  // EXPECT_EQ("Suburbs", idf_site.getString(Site_LocationFields::Terrain).get()); not in FT
```

Comment:
Terrain is on the Building Objects, should test for that

Reply:
The comment here is confusing.

But at least this is already tested here:

https://github.com/NREL/OpenStudio/blob/ba1b94b6542c1f1d4cdcdb775886852b9a82bc4f/src/energyplus/Test/Building_GTest.cpp#L97-L124